doc/attr.xml

```diff
@@ -1241,6 +1241,32 @@ gap> DigraphLoops(gr);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DigraphMycielskian">
+<ManSection>
+  <Attr Name="DigraphMycielskian" Arg="digraph"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>digraph</A> is a symmetric digraph, then <C>DigraphMycielskian</C> returns the Mycielskian of <A>digraph</A>. If <A>digraph</A> is not symmetric, then an error message is returned.<P/>
+    
+   The Mycielskian of a symmetric digraph is a larger symmetric digraph constructed from it, which has a larger chromatic number. For further information, see <URL> https://en.wikipedia.org/wiki/Mycielskian </URL>. <P/>
```

Comment:
Could you please have the `<URL>` tags tight against the URL, i.e.
`<URL>https://en.wikipedia.org/wiki/Mycielskian</URL>`